packages/text-input/text-input.scss

```diff
@@ -27,13 +27,31 @@ $textarea-height--expanded: $input-height * 7 + $bottom-padding;
     min-height: $input-total-height;
     padding-top: $label-height;
     background-color: $input-bg-color;
-    padding-bottom: $component-spacing--large;
 
     &--inline {
         display: inline-block;
         vertical-align: bottom;
     }
 
+    &__input:focus ~ .jkl-text-field__placeholder {
+        color: darkgray;
```

Comment:
Shouldn't this be one of our colors? And if its missing, maybe a FIXME or something to update it.